mautic/mautic
# Fix report with aggregated columns is not saved properly

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ Y ]
| New feature/enhancement? (use the a.x branch)      | [ N ]
| Deprecations?                          | [ N ]
| BC breaks? (use the c.x branch)        | [ N ]
| Automated tests included?              | [ N ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #11311 



#### Description:
This PR fixes issue with saving report with aggregated columns #11311



#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Go to reports and create a new report
3. Fill name field and select type Emails sent
4. Add column `ID` and group by `ID`. Add Two aggregated columns: SUM Read, SUM Bounce
![obraz](https://user-images.githubusercontent.com/8580942/177121850-60be4110-eeda-4f6d-92f6-2fc3d7edc063.png)
5. Click Save & Close
6. Report should be saved successfully
